packages/server/src/synchronizer/index.ts

```diff
@@ -58,6 +59,28 @@ export async function synchronizeFiles({
 
   await clean(destPath)
 
+  const allRoutes = entries.filter((route) => route.includes('routes'))
```

Comment:
Routes folder can be `routes` or `pages`

Reply:
Is it _either_ routes or pages, or could someone have both `routes` and `pages` in the same app?

Reply:
I think we should abandon supporting both. It's added complexity in our code that we don't have to have since we are a greenfield project :)

Note, we are changing back to `pages` from `routes` 😏 #107

Reply:
Okay cool! For now should I just leave as is?

Reply:
Hmm good question. At this point I am making the assumption it could be both in the same app. Perhaps you have a legacy next app and you want your blitz modules to use `routes`. If we disallow one we should have a warning saying the user has accidentally used the other.  I kind of think it is better to just allow both as it helps when people are refactoring although I can see an argument for making it one or the other.

Reply:
Ok that's great @flybayer I will adjust the pages rules.

Reply:
@flybayer Up to you! 😄

Reply:
Yeah, let's only support `pages`. So @merelinguist you can change this to be pages.

The change in #107 affects two things here:
- Now also need to detect duplicates for `api` directories scattered throughout the app
- Need to warn the developer if they try to use `pages/api`